### PR TITLE
Unblacklist recipe for pantools

### PR DESCRIPTION
**Pull request by npavlovikj**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by npavlovikj** [comment]
23:24:22 BIOCONDA ERROR STDOUT+STDERR:
/home/circleci/project/miniconda/bin/involucro -v=2 -f /home/circleci/project/miniconda/lib/python3.6/site-packages/galaxy/tools/deps/mulled/invfile.lua -set CHANNELS='conda-forge,file:///home/circleci/project/miniconda/conda-bld,bioconda,defaults' -set TARGETS='pantools=1.2=0' -set REPO='quay.io/biocontainers/pantools:1.2--0' -set BINDS='build/dist:/usr/local/,/home/circleci/project/miniconda/conda-bld:/home/circleci/project/miniconda/conda-bld' -set CONDA_IMAGE='quay.io/dpryan79/mulled_container:latest' -set TEST='pantools help 2>&1 | grep "PanTools" > /dev/null' build-and-test
**.Passing program arguments as a string may be a security hazard if combined with untrusted input**

[May  5 23:23:24] STEP Run image [quay.io/biocontainers/pantools:1.2--0] with command [[pantools help 2>&1 | grep "PanTools" > /dev/null]]
[May  5 23:23:24] DEBU Creating container [step-3ab7c35e83]
[May  5 23:23:24] DEBU Created container [d14d7e4d5647 step-3ab7c35e83], starting it
[May  5 23:23:24] DEBU Container [d14d7e4d5647 step-3ab7c35e83] started, waiting for completion
**[May  5 23:23:24] ERRO Task processing failed: Unexpected exit code [1] of container [d14d7e4d5647 step-3ab7c35e83], container preserved**

File: recipes/pantools/pantools.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/anaconda1anaconda2anaconda3/bin/python
```
Comment (epruesse) [inline recipes/pantools/pantools.py]: Hmm. I see why you'd want to do this, but I'm not sure about the `anaconda1...` string. Is that one documented somewhere, or does it just happen to work at the moment?

File: recipes/pantools/meta.yaml [hunk 2]
```diff
@@ -1,31 +1,36 @@
 {% set name = "pantools" %}
-{% set version = "1.0" %}
-# Do not forget to update the version string in the pantools.py file
```
Comment (epruesse) [inline recipes/pantools/meta.yaml]: Does that not have to be done anymore? (I really just don't know)

File: recipes/pantools/build.sh [hunk 3]
```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
+
 set -eu -o pipefail
 
-outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
-mkdir -p $outdir
+PACKAGE_HOME=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
```
Comment (epruesse) [inline recipes/pantools/build.sh]: It's not really important, but `/opt/pkg-1.2.3` is usually for things that bring their own `/bin`/`lib`/etc structure and are isolated from the system. `/share/pkg/` is just fine for noarch data such as jars. `/lib/pkg/` would be as well, actually, it's where architecture dependent stuff would go.


YI, but IMO `share/PKG_NAME/` is just fine,

File: recipes/pantools/pantools.py (hunk 1 above)
Comment (npavlovikj) [reply recipes/pantools/pantools.py]: It does work, and in https://readthedocs.com/projects/continuumio-conda/downloads/pdf/latest/ `/opt/anaconda1anaconda2anaconda3` is still used as the default placeholder...

File: recipes/pantools/meta.yaml (hunk 2 above)
Comment (npavlovikj) [reply recipes/pantools/meta.yaml]: I don't know either - I didn't see version string in pantools.py, so  I assume not? I will remove that line in the new commit.

File: recipes/pantools/pantools.py (hunk 1 above)
Comment (epruesse) [reply recipes/pantools/pantools.py]: Ok. Looks documented then.

**Comment by npavlovikj** [comment]
Thanks for removing `$PKG_BUILDNUM` Elmar - I saw some other java recipes were using it, so I added it here...

**Comment by epruesse** [comment]
Well. It's not important, but really not necessary. It's customary for `/opt` because you yourself might be installing a new version there, and that way you can switch back and forth just setting a symlink.

**Comment by npavlovikj** [comment]
@BiocondaBot merge

**Comment by npavlovikj** [comment]
@epruesse , https://github.com/bioconda/bioconda-utils/issues/508 implies that the merging user can not be the PR owner ("check that merging user is not PR owner"). Since I was able to merge this PR, is that feature temporarily turned off, or as long as there is co-author on the PR, self-merging is ok with the bot?

**Comment by epruesse** [comment]
It's temporarily turned off. While core agrees we need to have 4-eyes, we want to collect community feedback first.

See:

https://github.com/bioconda/bioconda-utils/blob/451ed558636408e0c392ce157b0681624df6f5ca/bioconda_utils/bot/commands.py#L147-L151

```python
@command_routes.register("merge")
@permissions(member=True)
#@permissions(author=False, team="core")
async def command_merge(ghapi, issue_number, user, *_args):
    """Merge PR"""
```

**Pull request merged**